src/lib/utilities/get-single-attribute-for-event.ts

```diff
@@ -208,20 +215,20 @@ const getSummaryAttribute = (event: WorkflowEvent): SummaryAttribute => {
   return first;
 };
 
-export const getSummaryForEventGroup = ({
+export const getSummaryForEventGroup = async ({
   lastEvent,
-}: EventGroup): SummaryAttribute => {
-  return getSummaryAttribute(lastEvent);
+}: EventGroup): Promise<SummaryAttribute> => {
+  return await getSummaryAttribute(lastEvent);
```

Comment:
Is the addition of async/await here needed? `getSummaryAttribute` already returns `Promise<SummaryAttribute>`.

Reply:
You are absolutely correct

Reply:
Think we can remove the async [here](https://github.com/temporalio/ui/pull/1795/commits/910dc77d274b840979e8e393cc19c2f65c54ca2d#diff-42a5d42a3ea1e209202949d938fa2ca7271443c0279d04c2991e2cf844f8e0c5R218) too.